src/index.html

```diff
@@ -1,13 +1,496 @@
-    <title>Title</title>
+    <title>WITCH OF MOUNTAIN WINDS</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      rel="shortcut icon" 
+      href="./images/svg/logo-colour.svg" 
+      type="image/png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container header__content">
+        <div class="header__top">
+          <div class="header__menu">
+            <a href="#menu" class="icon icon__menu header__nav--link"></a>
+          </div>
+          <div class="header__logo">
+            <a href="#">
+              <div class="icon icon__logo"></div>
+            </a>
+          </div>
+          <ul class="header__nav">
+            <li>
+              <a href="#shop" class="header__nav--link">
+                shop
+              </a>
+            </li>
+            <li>
+              <a href="#contacts" class="header__nav--link">
+                contact
+              </a>
+            </li>
+            <li>
+              <a href="#about" class="header__nav--link">
+                about
+              </a>
+            </li>
+          </ul>
+          <div class="header__basket">
+            <a href="#basket" class="icon icon__basket"></a>
+          </div>
+        </div>
+        <div class="header__main grid">
+          <div class="header__main--image-container grid__item grid__item--desktop-7-12 grid__item--tablet-1-6">
+            <img
+            src="./images/png/1.1.png"
+            alt="header picture"
+            class="header__main--image"
+            >
+          </div>
+          <div class="header__main--info grid__item grid__item--desktop-2-6 grid__item--tablet-2-5">
+            <h1 class="header__main--title">
+              Brand of eco-cosmetics and magic candles
+            </h1>
+            <p class="header__main--paragraph">
+              20% of sales at a wild bird shelter
+            </p>
+            <a href="#shop" class="header__main--button button">
+              shop
+            </a>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <a href="#" class="menu__icon icon icon__cross"></a>
+          <ul class="menu__list">
+            <li class="menu__list--item">
+              <a href="#shop" class="menu__list--link">
+                shop
+              </a>
+            </li>
+            <li class="menu__list--item">
+              <a href="#contacts" class="menu__list--link">
```

Comment:
According to BEM methodology, you should use double dashes for modifiers. Change 'menu__list--link' to 'menu__list-link'.